packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -59,6 +62,27 @@ function TokenLogoFallback() {
   )
 }
 
+function BlockExplorerTokenLink({
+  chain,
+  address
+}: {
+  chain: Chain
+  address: string | undefined
+}) {
+  const addressShortened =
```

Comment:
nit: wouldn't it be better to `return null` on undefined address?

This way we don't have to check for the address when shortening and we also don't return an empty string with href leading to `/token/undefined`.